src/app/containers/FrontPageSection/__snapshots__/index.test.jsx.snap

```diff
@@ -226,6 +226,7 @@ exports[`FrontPageSection Container snapshots should render correctly for canoni
   -webkit-text-decoration: none;
   text-decoration: none;
   overflow-wrap: break-word;
+  overflow-wrap: break-word;
```

Comment:
@bbc/psammead-story-promo is being updated in https://github.com/bbc/simorgh/pull/6313 so that we dot have two definitions of the `overflow-wrap` property.

Reply:
good spot, will remove 👍